backend/Dockerfile

```diff
@@ -1,11 +1,28 @@
 FROM ubuntu:18.04
 
 RUN apt-get update && apt-get install -y python3 python3-pip libpq-dev gettext python-dev
-WORKDIR /match4everyone-backend
-COPY requirements.txt /match4everyone-backend/requirements.txt
+
+WORKDIR /backend
+COPY requirements.txt /backend/requirements.txt
 RUN pip3 install -r requirements.txt
-COPY requirements.prod.txt /match4everyone-backend/requirements.prod.txt
+COPY requirements.prod.txt /backend/requirements.prod.txt
 RUN pip3 install -r requirements.prod.txt
 
+COPY . .
+
+# Set Django Settings to production, for the built docker image this will use whitenoise to collect the static files
+# Our app will refuse to run without a SECRET_KEY. collectstatic parses the configuration module and fails
+# if no key is set. To prevent this a dummy key is used. In production mode this needs to be supplied
+# using an env file in the docker-compose.yml
+ENV DJANGO_SETTINGS_MODULE="match4everyone.settings.production" SECRET_KEY="CzXG4ItUiwLUfTH2abQQ0qTzMSRiiDni"
```

Comment:
We can't use dev here, because then the upload post processing with whitenoise will not run.
Will evaluate switching to whitenoise in dev, I think that should be ok as well, because runserver won't require collectstatic but need to verify.

Reply:
@maltezacharias We should also add an explanation to the readme

Reply:
I moved whitenoise to common configuration. It is thus no longer necessary to set this here and further documentation is not required.